Unable to build the included sample vulnerable project

```
$ git rev-parse HEAD
63342aff3de82b3ad7789911605e56f579593569
```

```
$ mvn clean compile jar:jar -U
...
[WARNING] The POM for com.redhat.victims:enforce-victims-rule:jar:1.3.4-SNAPSHOT is missing, no dependency information available
Downloading: http://repository.apache.org/snapshots/com/redhat/victims/enforce-victims-rule/1.3.4-SNAPSHOT/enforce-victims-rule-1.3.4-SNAPSHOT.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.984 s
[INFO] Finished at: 2017-12-11T12:42:30+05:30
[INFO] Final Memory: 12M/293M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:1.3.1:enforce (enforce-victims-rule) on project sample: Execution enforce-victims-rule of goal org.apache.maven.plugins:maven-enforcer-plugin:1.3.1:enforce failed: Plugin org.apache.maven.plugins:maven-enforcer-plugin:1.3.1 or one of its dependencies could not be resolved: Could not find artifact com.redhat.victims:enforce-victims-rule:jar:1.3.4-SNAPSHOT in apache.snapshots (http://repository.apache.org/snapshots) -> [Help 1]
```

The key error message is `Could not find artifact com.redhat.victims:enforce-victims-rule:jar:1.3.4-SNAPSHOT in apache.snapshots`.

Here is a patch to fix this build problem,

``` diff
diff --git a/sample/pom.xml b/sample/pom.xml
index d2f161f..d31d429 100644
--- a/sample/pom.xml
+++ b/sample/pom.xml
@@ -49,7 +49,7 @@
           <dependency>
             <groupId>com.redhat.victims</groupId>
             <artifactId>enforce-victims-rule</artifactId>
-            <version>1.3.4-SNAPSHOT</version>
+            <version>1.3.4</version>
           </dependency>
         </dependencies>
         <executions>
```

--- a/sample/pom.xml
+++ b/sample/pom.xml
@@ -49,7 +49,7 @@
           <dependency>
             <groupId>com.redhat.victims</groupId>
             <artifactId>enforce-victims-rule</artifactId>
-            <version>1.3.4-SNAPSHOT</version>
+            <version>1.3.4</version>
           </dependency>
         </dependencies>
         <executions>

